recipes/leftbar/style/leftbar_row.less

```diff
@@ -33,6 +33,14 @@
   //  ----------------------------------------------------------------------------
   position: relative;
   opacity: var(--leftbar-row-opacity);
+  display: flex;
+  background-color: var(--leftbar-row-color-background);
+  border-radius: var(--leftbar-row-radius);
```

Comment:
We seem to have lost row focus styling.

before:
![Screenshot 2023-06-15 at 3 08 23 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/c379b249-fea7-4175-8dfb-de92171de296)

after:
![Screenshot 2023-06-15 at 3 08 29 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/fb954a26-fa9a-492c-8f27-b422e2cfc82b)

Reply:
Fixed